boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -233,13 +233,54 @@ To program M7, an i.MX container image ``flash.bin`` must be made, which contain
 multiple elements required, like ELE+V2X firmware, System Manager, TCM OEI, Cortex-M7
 image and so on.
 
-The steps making flash.bin and programming should refer to ``Getting Started with
+Two methods to build and program ``flash.bin``.
+
+1. If ``CONFIG_BOARD_NXP_SPSDK_IMAGE`` is not available for the board variant,
+the steps making flash.bin and programming should refer to ``Getting Started with
 MCUXpresso SDK for IMX95LPD5EVK-19.pdf`` in i.MX95 `MCUX SDK release`_. Note that
 for the DDR variant, one should use the Makefile targets containing the ``ddr`` keyword.
-
 See ``4.2 Run an example application``, just rename ``zephyr.bin`` to ``m7_image.bin``
 to make flash.bin and program to SD/eMMC.
 
+2. If ``CONFIG_BOARD_NXP_SPSDK_IMAGE`` is available for the board variant, enable it and
+prepare dependency firmware images manually to build.
+For board ``imx95_evk/mimx9596/m7``, at least below firmware images are needed. (They could
+be found and built from i.MX Linux BSP release. west blobs to fetch will be supported.)
+
+.. code-block:: none
+
+   modules/hal/nxp/zephyr/blobs/imx-firmware/
+   └── imx95-19x19-lpddr5-evk
+       ├── imx-boot-imx95-19x19-lpddr5-evk-sd.bin-flash_all
+       ├── lpddr5_dmem_qb_v202311.bin
+       ├── lpddr5_dmem_v202311.bin
+       ├── lpddr5_imem_qb_v202311.bin
+       ├── lpddr5_imem_v202311.bin
+       ├── m33_image-mx95alt.bin
+       ├── mx95a0-ahab-container.img
+       ├── oei-m33-ddr.bin
+       └── oei-m33-tcm.bin
+
```

Comment:
I know currently we don't provide blobs to boot u-boot or Linux together with m-core u-boot simulteneously, but according to discussion in https://github.com/zephyrproject-rtos/zephyr/issues/82560, may I know whether can provide some document information  for how to do it?

Reply:
For this PR, I think multicore case is another topic/work to do, as all we did now in zephyr is only A55 or M7 independently. I can't provide that as I haven't done any verification on that.

But that's important to support multicore case. The difficulty is the permission/resource allocation to M7 and A55. It's case specific.

1. Permission/resource allocation to M7 and A55 must be exclusive.
2. Resource usage on M core Zephyr, and A core Linux BSP must be exclusive.

I don't think we can provide any generic support for multicore. But that will be very good, if we can provide a specific sample to demo multicore work together with zephyr in the future.

Thanks.

Reply:
I understand the resource and permission allocation is case by case, but maybe can provide a very simple hello world application which use almost so less hardware resource except,memroy, UART and some Core related hardware resource, but it will cover update in Linux dts, I am okay to provides such features in future, thanks.